index.html

```diff
@@ -0,0 +1,106 @@
+<!-- Author: Nicolas Caceres -->
+<!DOCTYPE html>
+<html lang="es">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CV Nicolás Cáceres</title>
+</head>
+
+<body>
+    <header>
+        <h1>Nicolas Joel Caceres Parra</h1>
+        <h2>Ingeniero Industrial e Ingeniero Informático</h2>
+        <img src="Perfil CV.jpg" alt="Profile Image">
+        <h3>Contacto</h3>
+        <ul>
+            <li>Teléfono: (+57) 318 418 0450</li>
+            <li>Dirección: Chía, Cundinamarca.</li>
+            <li>Correo: nicolascacpa@unisabana.edu.co</li>
+        </ul>
+    </header>
+
+    <main>
+        <h2>Sobre mí</h2>
+        <p>
+            Soy una persona que le gusta aprender e innovar, disfruto del conocimiento
+            y de conocer más acerca de las personas. Me interesan los temas de avances
+            tecnológicos, especialmente aquellos que involucran desarrollos en el sector energético
+            de fuentes sostenibles. Me caracterizo por mi personalidad amable y colaboradora.
+        </p>
+
+        <h2>Educación</h2>
+        <ul>
+            <li>
+                <h3>2011 - 2022: SummerHill School</h3>
+                <p>
+                    Bachiller académico graduado con honores como mejor bachiller
+                </p>
+            </li>
```

Comment:
```suggestion
            <li>
                <h3>2011 - 2022: SummerHill School</h3>
                <ul>
                   <li>
                       <p> Bachiller académico graduado con honores como mejor bachiller</p>
                    </li>
                </ul>
            </li>
```

De esta manera conserva el estilo de lista de listas.

*aplica para todos los elementos de este listado